ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -110,8 +110,8 @@ class TabBarItemView: UIView {
         if isInPortraitMode {
             container.axis = .vertical
             container.spacing = spacingVertical
-            imageHeightConstraint?.constant = portraitImageSize
-            imageWidthConstraint?.constant = portraitImageSize
+            imageHeightConstraint?.constant = titleLabel.isHidden ? portraitImageSize : landscapeImageSize
```

Comment:
Since we're now using "landscapeImageSize" in portrait, the name "landscapeImageSize" is now misleading. Maybe we can rename it to something else to reflect how it is different rather than when it is used?